src/bp-forums/admin/admin.php

```diff
@@ -460,8 +460,8 @@ public function admin_bar_about_link( $wp_admin_bar ) {
 					array(
 						'parent' => 'wp-logo',
 						'id'     => 'bbp-about',
-						'title'  => esc_html__( 'About Forums', 'buddyboss' ),
-						'href'   => add_query_arg( array( 'page' => 'bbp-about' ), admin_url( 'index.php' ) ),
+						'title'  => esc_html__( 'About BuddyBoss', 'buddyboss' ),
```

Comment:
@kishanjasani 
1. function docs need to update according to functionality 
2. Can you explain why we need the `hello` query param.

Reply:
This page is not available on the BuddyBoss Platform. It's a BuddyPress plugin page.

I add a hello param because we use it when we install the BuddyBoss plugin. It has a description video of the plugin.